apps/studio/components/interfaces/SQLEditor/AiAssistantPanel/AiMessagePre.tsx

```diff
@@ -20,6 +20,7 @@ interface AiMessagePreProps {
   className?: string
 }
 
+// [DEPRECATED] To delete
```

Comment:
for all of these if you use this syntax vs code will understand and make it clearer not to use them. Not needed if these are gonna be immediately deleted
```suggestion
/**
 * @deprecated to delete
 */
```

Reply:
yup! will look into removing these guys next week after LW :)